### ISSUE TITLE
Is there any non-stationary noise in interspeech2020/master?

### ISSUE THREAD

**Issue by st3inum** [facebookresearch/denoiser#119]
I was trying to reproduce the results of the DNS dataset [interspeech2020/master](https://github.com/microsoft/DNS-Challenge/tree/interspeech2020/master) dataset(dataset is generated with the default [configuration](https://github.com/microsoft/DNS-Challenge/blob/interspeech2020/master/noisyspeech_synthesizer.cfg)). But it seems that in the case of non-stationary data it works really bad, although the author's code/weight works correctly. I suspect that there is any non-stationary noise in interspeech2020 or not. Or am I doing something wrong in the configuration file? Should I try something else to get better results with non-stationary data?

**Comment by st3inum** [comment]
Yeah, I listened to some of the samples generated by the DNS script. But didn't find my expected noise although found some good quality stationary noise. 

By non-stationary noise, I meant to sound like [hammer hitting noise](https://www.youtube.com/watch?v=xpxgeOkiRnw)

**Comment by adiyoss** [comment]
Hi @st3inum,
Yes, there are plenty of non-stationary noises in DNS. You have hammering noise as you shared, babies crying, dogs barking, etc.
